File: LottieSample/src/main/java/com/airbnb/lottie/samples/ILottieApplication.java
package com.airbnb.lottie.samples;

import android.support.v4.util.Pair;

interface ILottieApplication {
  void startRecordingDroppedFrames();

  /**
   * Returns the number of frames dropped since starting
   **/
  Pair<Integer, Long> stopRecordingDroppedFrames();
}

File: LottieSample/src/main/java/com/airbnb/lottie/samples/ILottieApplication.kt
package com.airbnb.lottie.samples

import android.support.v4.util.Pair

internal interface ILottieApplication {
    fun startRecordingDroppedFrames()

    /**
     * Returns the number of frames dropped since starting
     */
    fun stopRecordingDroppedFrames(): Pair<Int, Long>
}
